vibi-dpu/src/pubsub/listener.rs

```diff
@@ -60,12 +62,46 @@ async fn process_message(attributes: &HashMap<String, String>, data_bytes: &Vec<
 			process_pat_repos(&data_bytes).await;
 			log::info!("Processed repos successfully");
 		}
+		"health_check" => {
+			log::info!("Sending Health Check Info to Vibinex...");
+			health_check().await;
+		}
 		_ => {
 			log::error!("[process_message] Message type not found for message : {:?}", attributes);
 		}
 	};
 }
 
+async fn health_check() {
```

Comment:
LOW LOGAF: Can we put this function somewhere else? `listeners.rs` has grown quite large.